src/Servers/Connections.Abstractions/src/ConnectionContext.cs

```diff
@@ -12,22 +12,50 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// Encapsulates all information about an individual connection irrespective of the underlying transport.
+    /// </summary>
     public abstract class ConnectionContext : IAsyncDisposable
     {
+        /// <summary>
+        /// Gets or sets a unique identifier to represent this connection in trace logs.
+        /// </summary>
```

Comment:
`<value>` elements for these properties?

Reply:
We don't do it anywhere. We need to do a huge find replace and update all the properties. Also is it valuable to describe the gets part of gets and sets?

Reply:
I don't think waiting for "huge find replace" operations that never happen is worth it. Even so, it doesn't preclude the value of just fixing them when we find them.

Reply:
I don't see the "value" in adding them, can you point me some prior art?

Reply:
`<value>Using this element instead of 'returns' makes it actually show up in API docs both in IntelliSense and docs.microsoft.com</value>` :trollface: 

But seriously, compare one of our properties:

[![image](https://user-images.githubusercontent.com/7574/60037419-50ec3a00-9666-11e9-86ac-f99d45cadf3e.png)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.options.optionsmonitor-1.currentvalue?view=aspnetcore-3.0)

With one of the BCL ones:

[![image](https://user-images.githubusercontent.com/7574/60037462-6feacc00-9666-11e9-8a2f-0fd2ab2c9222.png)](https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocketoptions.clientcertificates?view=netcore-3.0#System_Net_WebSockets_ClientWebSocketOptions_ClientCertificates)

If we think that it doesn't really matter to fill in the "Property Value" heading (because the summary is sufficiently explaining the property) then I'm **fine** with that, but the `<returns>` element does nothing.

So, I guess I'm kinda responding more to the other one (where you did put a `<returns>`). If you don't want to do it, let's just not do it, rather than half doing it (by using `<returns>`, which doesn't work, instead of `<value>`, which does).

Reply:
> So, I guess I'm kinda responding more to the other one (where you did put a <returns>). If you don't want to do it, let's just not do it, rather than half doing it (by using <returns>, which doesn't work, instead of <value>, which does).

Sounds good to me.